test/dotcom_web/live/trip_planner_test.exs

```diff
@@ -65,18 +65,6 @@ defmodule DotcomWeb.Live.TripPlannerTest do
     end
 
     test "pushes updated location to the map", %{view: view} do
-      updated_location = %{
```

Comment:
Should we just delete the test, rather than leaving it empty?

Reply:
Does it matter? Leaving it there signals we're going to come back and do this later.

Reply:
👍 I didn't get that that was the signal. I'm not used to dropping tests and re-adding them after the fact, so I didn't get that that's what was happening here.